rebasebot/cli.py

```diff
@@ -181,6 +181,20 @@ def _parse_cli_arguments(testing_args=None):
         choices=["none", "soft", "strict"],
         help="Option that shows how to handle UPSTREAM tags in "
              "commit messages. (default: %(default)s)")
+    parser.add_argument(
+        "--allow-bot-squash",
+        action="store_true",
+        default=False,
+        required=False,
+        help="When enabled, the bot will be able to squash some bot's commits - provided a list of bot emails.",
+    )
+    parser.add_argument(
+        "--bot-emails",
+        type=str,
+        nargs="+",
+        required=False,
+        help="Specify the bot emails to be able to squash their commits.",
+    )
```

Comment:
How do I specify more than one email if this is just a string? Can I use comma separated values or?

Reply:
You can do: `--bot-emails email1 email2`.